app/models/polymorphic/task.rb

```diff
@@ -104,7 +104,8 @@ class Task < ActiveRecord::Base
   }
 
   acts_as_commentable
-  has_paper_trail class_name: 'Version', meta: { related: :asset }, ignore: [ :subscribed_users ]
+  has_paper_trail class_name: 'Version', meta: { related: :asset },
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.